scripts/qemu.py

```diff
@@ -86,12 +86,12 @@ def _remove_if_exists(path):
             raise
 
     def is_running(self):
-        return self._popen and (self._popen.returncode is None)
+        return self._popen and (self._popen.poll() is None)
```

Comment:
This includes the regression I mentioned in email:

```
    >>> import qemu, os
    >>> a=qemu.QEMUMachine("/usr/bin/qemu-kvm", debug=True)
    >>> a.launch()
    QMP:>>> {'execute': 'qmp_capabilities'}
    QMP:<<< {u'return': {}}
    >>> os.kill(a.get_pid(), 9)
    >>> a.is_running()
    False
    >>> a.shutdown()
    >>> os.path.exists(a._qemu_log_path)
    True
```

It is addressed by the last commit so I suggest just reorganizing those patches.